acisfp_check/acisfp_check.py

```diff
@@ -492,6 +539,35 @@ def search_obsids_for_viols(self, limit_name, limit, observations, temp, times,
         # Finished - return the violations list
         return viols_list
 
+    def write_temps(self, outdir, times, temps):
+        """
+        Write the states record array to the file "temperatures.dat".
+
+        Parameters
+        ----------
+        outdir : string
+            The directory the file will be written to.
+        times : NumPy array
+            Times in seconds from the start of the mission
+        temps : NumPy array
+            Temperatures in Celsius
+        """
+        outfile = os.path.join(outdir, 'temperatures.dat')
+        mylog.info('Writing temperatures to %s' % outfile)
+        T = temps[self.name]
+        e = self.predict_model.comp['earthheat__fptemp'].dvals
+        temp_recs = [(times[i], DateTime(times[i]).date, T[i], e[i])
+                     for i in range(len(times))]
+        temp_array = np.rec.fromrecords(
+            temp_recs, names=('time', 'date', self.msid, 'earth_fov'))
+        fmt = {self.msid: '%.2f',
+               'time': '%.2f',
+               'earth_solid_angle': '%.7f'}
+        out = open(outfile, 'w')
+        Ska.Numpy.pprint(temp_array, fmt, out)
```

Comment:
I don't have a problem with this, but I've been moving to using astropy.table for all of our table printing needs.

Reply:
This is just a slight modification to what the default acis_thermal_check code is already doing, which is why it uses the old table writing. We can probably switch to AstroPy tables at some point in the near futurez

Reply:
This is a deviation from the standard format of temperatures.dat files that has been in existence for years. And it's a deviation that's specific to focal plane temperature model temperatures.dat files. So lots of people are not expecting this.

I don't have any problem with that per se.  But we have to be sure everyone who might use the files in other code are aware of the change.  We want to minimize the chances someone's code that, heretofore has worked, suddenly stops working. Given that this is a new column placed on the end, the chances are minimized that something will break but not zero.

Reply:
The PSMC temperatures.dat file used to have pin1at in there, so this isn’t a completely new thing. 

I am pretty sure it is only us and SOT MP that care about this file so I will make sure to alert them to this.

Reply:
Plan B might be a new file `earth_solid_angles.dat`.  Then SOT MP would not need to change their code (assuming that is the case now?).

Reply:
We don't know that SOT MP has code that would need to be changed. I'm just raising the issue so it gets thought over.